cam2/src/main/java/com/commonsware/cwac/cam2/ConfirmationFragment.java

```diff
@@ -201,22 +210,30 @@ private void loadImage(Float quality) {
     if (getArguments().getFloat(ARG_FACE_OCCUPANCY) > 0) {
       OccupancyResult occupancyResult = faceOccupancyDetector.isFacePresentWithMinimumOccupancy(
           imageContext.getContext(), bitmap, getArguments().getFloat(ARG_FACE_OCCUPANCY));
-      facePass = false;
       if (occupancyResult == OccupancyResult.NO_FACE) {
-        showRetryOption(getString(R.string.error_no_face));
-        return;
+        if (!facePass) {
+          facePass = true;
```

Comment:
this is because, if it detected face is not close enough , it means user is already taking face photo. so no need to show that error again